user-guide/components/SubReport/creating-nested-hierarchy-with-subreports.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Creating Nested Hierarchy
+page_title: Creating Nested Hierarchy
+description: Learn how to use SubReports in the Web Report Designer to create nested hierarchy.
+slug: wrd-user-guide-create-nested-hierarchy-with-subreport
+tags: web, report, design, components, subreport, hierarchy, master, detail, nested
+published: True
+reportingArea: WRDHTML5, WRDBlazorWrapper
+position: 2
+---
+
+# Creating Nested Hierarchy with SubReports
+
+The Web Report Designer provides an alternative approach to parameter binding for hierarchical data using the **DataSource** property of the **SubReport** item. This method is cleaner when your detail report is fully data-bound (not just a table inside it) and works well for hierarchical nested data scenarios.
```

Comment:
As discussed, consider moving these two sentences to the Subreport Overview and start this article with the line "Using DataSource on the SubReport..."